README.md

```diff
@@ -146,43 +146,41 @@ readData(dataInput) {
 ```
 
 
-# Development
+# Development and Testing
 
-## Building And Installing from Sources
+Hazelcast Node.js client is developed using Typescript. If you want to help with bug fixes, develop new features or
+tweak the implementation to your application's needs, you can follow the steps in this section.
 
-Follow the below steps to build and install Hazelcast Node.js client from its source:
-
-- Clone the GitHub repository [https://github.com/hazelcast/hazelcast-nodejs-client.git](https://github.com/hazelcast/hazelcast-nodejs-client.git).
-- Install the dependencies using the command `npm install`.
-- Compile TypeScript using the command `npm run compile`.
-- Link the package locally using the command `npm link`.
-- Run static analysis tool using the command `npm run lint`
+## Building and Using Client From Sources
 
-## Using Locally Installed Package
+Follow the below steps to build and install Hazelcast Node.js client from its source:
```

Comment:
...install Hazelcast Node.js Client..

Reply:
@Serdaro advised to be consistent and use `Hazelcast Node.js client` in one of the other PRs. I could not find his specific comment but that is what I remember. Maybe he can confirm.

Reply:
I was using as `Hazelcast Node.js Client`. @Serdaro Can we clarify?

Reply:
@furkansenharputlu , most of the "client" in this Readme is expressed as "client" which is also a convention in out other public facing documents. So: "Hazelcast Node.js client"